microsoft-edge/visual-studio-code/index.md

```diff
@@ -38,3 +38,13 @@ By adding the [Elements for Microsoft Edge](https://marketplace.visualstudio.com
 For more information, check out [this page](./elements-for-edge.md).
 
 ![GIF of the Elements for Edge VS Code extension in action!](./media/elements-for-edge.gif)
+
+## webhint
+
+[webhint](https://webhint.io) is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors. This open-source project, initially developed by the Microsoft Edge team, is now part of the [OpenJS Foundation](https://openjsf.org/). The Microsoft Edge team continues to contribute to webhint alongside web developers in the community.
+
+![GIF of webhint VS Code extension](./media/webhint-extension.png)
+
+By adding the [webhint extension for VS Code](https://marketplace.visualstudio.com/items?itemName=webhint.vscode-webhint), you can identify and fix problems in your HTML, CSS, JavaScript, TypeScript, and more. Hints appear as inline underlines and are summarized in the Problems pane.
```

Comment:
"By adding the [webhint extension for VS Code](https://marketplace.visualstudio.com/items?itemName=webhint.vscode-webhint), you can identify and fix problems in your HTML, CSS, JavaScript, TypeScript, and more. Hints appear as inline underlines and are summarized in the Problems pane."
to
"Identify and fix problems in your HTML, CSS, JavaScript, TypeScript, and more by adding the [webhint extension for VS Code](https://marketplace.visualstudio.com/items?itemName=webhint.vscode-webhint).  Hints appear as inline underlines and are summarized in the Problems pane.".

Reply:
Done